kfdefs/overlays/moc/smaug/opf-jupyterhub-stage/kustomization.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: opf-jupyterhub-stage
+
+resources:
+  - ../../../../base/jupyterhub
```

Comment:
can you also copy the pvc's onto here same as the one for zero

Reply:
no we don't need the PVCs here